src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -443,11 +443,15 @@ When using the `<ViewTransitions />` router, the following steps occur to produc
 7. The router executes any new scripts added to the page.
 8. The `astro:page-load` event fires. This is the end of the navigation process.
 
-## Script behavior during page navigation
+## Script behavior
```

Comment:
```suggestion
## Script behavior with view transitions
```
Do you have a reason for removing this? This original title attempted to disambiguate for people not using view transitions, so they wouldn't necessarily end up on this heading when doing a search. Perhaps this is better?